src/utils/cta-utils.ts

```diff
@@ -50,3 +53,39 @@ export function showAlreadyOptedInToast(
   }
   new Toast(deps, feUrl('/toastiframe', urlParams)).open();
 }
+
+/**
+ * Gets the complete contribution URL that should be used for the activity iFrame view.
+ */
+export function getContributionsUrl(
+  clientConfig: ClientConfig,
+  clientConfigManager: ClientConfigManager,
+  pageConfig: PageConfig,
+  isInlineCta: boolean = false
+): string {
+  if (!clientConfig.useUpdatedOfferFlows) {
+    return feUrl('/contributionsiframe');
+  }
+
+  if (clientConfigManager.shouldForceLangInIframes()) {
```

Comment:
Can we simplify this section with something like:
```suggestion
  params = {'publicationId': pageConfig.getPublicationId()};
  if (clientConfigManager.shouldForceLangInIframes()) {
    params['hl'] = clientConfigManager.getLanguage();
  }
  if (isInlineCta) {
    params['ctaMode'] = 'CTA_MODE_INLINE';
  }
  return feUrl(/contributionoffersiframe', params);
```